site/js/event.js

```diff
@@ -92,43 +91,45 @@ function renderEvents() {
     if (today_year < start_detail[2]) event_status = "Registration Open";
     else if (today_year == start_detail[2] && today_month + 1 < start_detail[1])
       event_status = "Registration Open";
-    else if (
-      today_year == start_detail[2] &&
-      today_month + 1 == start_detail[1] &&
-      today_date <= start_detail[0]
-    )
+    else if (today_year == start_detail[2] && (today_month + 1) == start_detail[1] && today_date <= start_detail[0])
       event_status = "Registration Open";
 
-    // Set the href based on status
-    if (event_status !== "Completed") {
+    // If event is not complete, refer to registration for the event
+    // otherwise, refer to the gallery section of the corresponding event
+    if (event_status != "Completed") {
       title.href = elt.URL;
     } else {
-      title.href = elt.IMGURL || elt.URL;
+      if (elt.IMGURL) title.href = elt.IMGURL;
+      else title.href = elt.URL;
     }
 
+    title.innerText = elt.Title;
     title.classList.add("event-title");
-
-    if (event_status === "Completed") {
-      status.classList.add("event-status", "completed");
-    } else if (event_status === "Registration Open") {
-      status.classList.add("event-status", "registration-open");
```

Comment:
Don't remove the addition of class "registration-open"

This also beautifies the text